packages/wdio-cucumber-framework/src/cucumberFormatter.ts

```diff
@@ -112,6 +113,10 @@ export default class CucumberFormatter extends Formatter {
         this.failAmbiguousDefinitions = options.parsedArgvOptions._failAmbiguousDefinitions
     }
 
+    normalizeURI(uri?: string) {
+        return uri ? (path.isAbsolute(uri) ? uri : path.resolve(uri)) : undefined
```

Comment:
`path.resolve(uri)` would fail if `wdio.conf.js` is not within the working directory right?

Reply:
I think it shouldn't result in failure because the URI we're attempting to resolve comes from the loaded spec files path.

Additionally performed a dry run by moving the `conf` file inside a dir and it passed.
 
![Screenshot 2023-09-11 at 1 12 56 PM](https://github.com/webdriverio/webdriverio/assets/49207189/256af9ee-4eb1-4ee9-9bf6-758d39f92e8b)

Reply:
Thanks for checking 👍